scapy/contrib/gtp.uts

```diff
@@ -23,9 +23,10 @@ gtp.dport == 2123 and gtp.teid == 2807 and len(gtp.IE_list) == 5
 = GTPCreatePDPContextRequest(), basic dissection
 ~ random_weird_py3
 random.seed(0x2807)
-raw(gtp)
-assert _ in [b"E\x00\x00K\x00\x01\x00\x00@\x11|\x9f\x7f\x00\x00\x01\x7f\x00\x00\x01\x08K\x08K\x007\x1a\xc30\x10\x00'\x00\x00\n\xf7\x10\xd6\xd2\xf6\xd8\x14\x0b\x85\x00\x04\xa3\xad\x98\xfa\x85\x00\x04F\\`\xd6\x87\x00\x0fBCD5lPP8N2u8h9l",
-             b"E\x00\x00K\x00\x01\x00\x00@\x11|\x9f\x7f\x00\x00\x01\x7f\x00\x00\x01\x08K\x08K\x007\xd3\xf20\x10\x00'\x00\x00\n\xf7\x107m\xaeV\x14\x08\x85\x00\x04\x83\x92K\xa3\x85\x00\x04\xb7\xb2\xff\xd0\x87\x00\x0foTrnmM9erqfhqpV"]
+rg = raw(gtp)
+print(rg)
```

Comment:
Is this a forgotten debug change?

Reply:
This is a bit unrelated, but that test failed on my Windows machine. To replicate the fact that it was printed, then checked, I added a print